netlify.toml

```diff
@@ -35,3 +36,7 @@
   from = "https://devto.netlify.com/*"
   to = "https://docs.dev.to/:splat"
   status = 301
+  [[redirects]]
```

Comment:
This is just an idea as I don't know how to deploy multiple directories to Netlify. As well the `storybook.dev.to` domain would need to be set up in Netlify.

Reply:
I believe there's a way, but I'll have to dig into their docs and figure it out. We can set up `storybook.dev.to` in Netlify as well.

In general, TOML/YAML have always felt pretty cryptic to me. :|

Reply:
I can dig in as I imagine you have a few things on your plate 😉

Reply:
Sure if you'd like. Would certainly be appreciated!

Reply:
Actually, I guess it depends how you slice it. Storybook is its own static site, so it probably makes more sense to have a new site in Netlify just for it. We could overwrite the netlify.toml file with one that is say called storybook.netlify.toml that gets renamed to netlify.toml. and then deploy.

I think that makes more sense than what is currently in this PR. Thoughts? @Zhao-Andy?

Reply:
Hmm I think that makes more sense, too. Like you said, I think it would have to be a new Netlify site, but we might be able to share the same netlify.toml file. Not sure the exact configuration so I'll have to look into it more. I'm okay with whatever you think is best.

Reply:
Cool beans. I'll pop up a change shortly and maybe discuss it with the rest of the team.